stable_baselines3/ppo/ppo.py

```diff
@@ -282,19 +276,24 @@ def train(self, n_epochs: int, batch_size: int = 64) -> None:
         explained_var = explained_variance(self.rollout_buffer.returns.flatten(),
                                            self.rollout_buffer.values.flatten())
 
-        logger.logkv("n_updates", self._n_updates)
-        logger.logkv("clip_fraction", np.mean(clip_fraction))
-        logger.logkv("clip_range", clip_range)
+        # Logs
+        logger.record("loss/entropy_loss", np.mean(entropy_losses))
+        logger.record("loss/policy_gradient_loss", np.mean(pg_losses))
+        logger.record("loss/value_loss", np.mean(value_losses))
+        logger.record("loss/approx_kl", np.mean(approx_kl_divs))
+        logger.record("loss/clip_fraction", np.mean(clip_fraction))
+        logger.record("loss/loss", loss.item())
+        logger.record("loss/explained_variance", explained_var)
+        if hasattr(self.policy, "log_std"):
+            logger.record("loss/std", th.exp(self.policy.log_std).mean().item())
+
+        logger.record("session/n_updates", self._n_updates, exclude="tensorboard")
+        logger.record("session/episode_reward", np.sum(self.rollout_buffer.rewards))
```

Comment:
you can remove that one (and it seems buggy, I tested it with the rl zoo using `python train.py --algo ppo --env CartPole-v1 -tb /tmp/sb3`)

Reply:
in fact, this one is not correct